src/ray/raylet/main.cc

```diff
@@ -21,7 +21,9 @@ DEFINE_string(config_list, "", "The raylet config list of this node.");
 DEFINE_string(python_worker_command, "", "Python worker command.");
 DEFINE_string(java_worker_command, "", "Java worker command.");
 DEFINE_string(redis_password, "", "The password of redis.");
-DEFINE_string(temp_dir, "", "Temporary directory.");
+DEFINE_string(session_dir, "", "Session directory.");
+DEFINE_string(debug_state_filename, "/tmp/ray/debug_state.txt",
```

Comment:
This string never actually gets used, right? Can we just make it `""` and in the third argument say `"The path of the debug state file. This will be something like /tmp/ray/session_*/debug_state.txt."`.